rules/windows/execution_from_unusual_path_cmdline.toml

```diff
@@ -0,0 +1,37 @@
+[metadata]
+creation_date = "2020/10/30"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/30"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies process execution from suspicious default Windows directories. This is sometimes done by adversaries to hide malware in trusted paths."
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution from Unusual Directory - CommandLine"
+risk_score = 47
```

Comment:
It may be useful to add a `note` that this is related to the other non-commandline version rule